packages/kit/src/vite/build/build_server.js

```diff
@@ -239,6 +239,8 @@ export async function build_server(options, client) {
 		/** @type {string[]} */
 		const imports = [];
 
+		// String representation of
+		/** @type {import('types').SSRNode} */
```

Comment:
this is kind of a confusing way to write the comment. maybe something like this?
```suggestion
		// String representation of types.SSRNode
```

Reply:
I specifically wrote it like this so that you can do "find usages" on the type and it takes you to this place. This would have been helpful to me to find the correct places I need to change quicker.